app/components/citations/multiple_citations_component.html.erb

```diff
@@ -9,21 +12,34 @@
       <% end %>
     </select>
   </div>
+  <% end %>
 
   <% Citations::MarcCitation::FORMATS.each.with_index do |format, index| %>
     <div class="bg-light p-3"<%= ' hidden' if index != 0 %> data-citation-format-target="panel"
       data-controller="copy-text" id="citation-format-<%= format %>">
       <div class="d-flex align-items-center gap-2 mb-2">
-        <h4 class="mb-0 me-3"><%= t format, scope: 'searchworks.citations.styles' %></h4>
+        <h4 class="mb-0 me-3">
+          <% if preferred_or_unavailable_citations_only? %>
+            Preferred citations
+          <% else %>
+            <%= t format, scope: 'searchworks.citations.styles' %>
+          <% end %>
+        </h4>
         <button class="btn btn-outline-primary" data-action="click->copy-text#copy">Copy</button>
       </div>
-      <div class="mb-3" data-copy-text-target="text">
+      <% if unavailable_citation_count(format).positive? %>
+        <div class="alert citations-alert bi bi-exclamation-circle-fill p-2 my-3"><span class="ms-2">Citations are unavailable for <%= unavailable_citation_count(format) %> of the <%= all_citations.count %> saved records.</span></div>
```

Comment:
Should we handle the `all_citations.count = 1` case  a little nicer?  (and maybe all citations being unavailable?)